blank_rust_project/src/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+  <style>
+    .App-header {
+      background-image: url('https://nearprotocol.com/wp-content/uploads/2019/03/illo-developers.svg');
+      min-height: 100vh;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      font-size: 1rem;
+      color: #25282A;
+      text-align: center;
+      padding:2%; 
+    }
+    .image-wrapper {
+      display: flex;
+      align-items: center;
+      flex-direction: column;
+      justify-content: center;
+    }
+    .App-logo {
+      animation: App-logo-spin infinite 20s linear;
+      pointer-events: none;
+    }
+
+    .margin-logo {
+      margin: 20px;
+    }
+
+    @keyframes App-logo-spin {
+      from {
+        transform: rotate(0deg);
+      }
+      to {
+        transform: rotate(360deg);
+      }
+    }
+
+    button {
+      padding: 5px;
+      color: #0072CE;
+      border-radius: 10px;
+      border: 3px solid #0072CE;
+    }
+
+    button:hover {
+      cursor: pointer;
+      color: white;
+      background: #0072CE;
+    } 
+
+    .speech {
+      font-size: 1.5rem;
+      color: #0072CE;
+      text-shadow: 1px 1px #D1CCBD;
+    }
+  </style>
+</head>
+<body class="App-header">
+  <div>
+      <div class="image-wrapper">
+        <img style="width:33%;" src="assets/gray_near_logo.svg" alt="NEAR logo" />
+        <p><span role="img" aria-label="fish">🐟</span> NEAR protocol is a new blockchain focused on developer productivity and useability!<span role="img" aria-label="fish">🐟</span></p>
+        <p><span role="img" aria-label="chain">⛓</span> This little react app is connected to blockchain right now. <span role="img" aria-label="chain">⛓</span></p>
+        <img src="assets/near.svg" class="App-logo margin-logo" alt="logo" />
+        <p id="speech" class="speech"></p>
+      </div>
+      <div id="signed-out-flow" class="d-none">
+        <button id="sign-in-button" >Sign-in with NEAR</button>
+      </div>
+      <div id="signed-in-flow" class="d-none">
+          <button id="sign-out-button" >Sign-out</button>
+      </div> 
+  </div>
+  <script src="https://cdn.jsdelivr.net/gh/nearprotocol/nearlib@0.19.1/dist/nearlib.js"></script>
```

Comment:
probably needs newer nearlib

Reply:
done. We need to update other projs too - I'll put that in a separate PR